### PR TITLE
Update README and corresponding artifacts

### PR DESCRIPTION
**Pull request by DilanUA**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,37 @@ Kubernetes Artifacts for
 Container-based Deployment Patterns
 of WSO2 Enterprise Integrator
 
-This project is still under development.
\ No newline at end of file
+This initial version (V0.1) contains the deployment of a single integrator container instance
```
Comment (imesh) [inline README.md]: I think it's better not to mention the version here. Release note might be the best place to mention this.

File: README.md [hunk 2]
```diff
@@ -3,4 +3,32 @@ Kubernetes Artifacts for
...
-This project is still under development.
\ No newline at end of file
+This initial version (V0.1) contains the deployment of a single integrator container instance 
+running with one local [h2] registry database and one external [MySQL] user management database.
+ 
+## Getting Started
+
+>In the context of this document, `KUBERNETES_HOME` will refer to a local copy of [`wso2/kubernetes-ei artifacts`](https://github.com/wso2/kubernetes-ei/) and you have to have git, docker and Kubernetes client, kubectl installed in your local machine to execute following steps.
+
+##### 1. Checkout WSO2 kubernetes-ei repository using `git clone`
```
Comment (imesh) [inline README.md]: Might be better to add a colon at the end of this sentence

File: README.md (hunk 1 above)
Comment (DilanUA) [inline README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (DilanUA) [inline README.md]: Fixed.

**Pull request merged**